content/discrete-math/concepts/congruences/congruences.md

```diff
@@ -0,0 +1,86 @@
+---
+Title: 'Congruences'
+Description: 'Congruences refer to the relationship between two integers a and b when they have the same remainder after
+division by positive integer m that is greater than 1.'
+Subjects:
+  - 'Computer Science'
+  - 'Discrete Math'
+Tags:
+  - 'Arithmetic'
+  - 'Math'
+  - 'Logical'
+  - 'Logic'
+CatalogContent:
+  - 'computer-science'
+  - 'paths/computer-science'
+---
+
+**Congruences** refer to the relationship between two integers, _a_ and _b_, that have the same remainder after division
+by positive integer, _m_, that is greater than 1.
+
+## _a_ divides _b_
+
+If _a_ and _b_ are integers with a != 0 then _a_ divides _b_ if an integer _c_, exists given that _b = ac_.
```

Comment:
```suggestion
If _a_ and _b_ are integers with a ≠ 0 then _a_ divides _b_ if an integer _c_, exists given that _b = ac_.
```

Reply:
I wonder if just using the symbol will work

Reply:
Sure. It forgot that it previews here then it should in case of preview as well. Making the changes.